src/components/IssueListItem.js

```diff
@@ -41,9 +41,11 @@ function IssueListItem({title, labels, author, opened, type, participants, comme
                   {milestone.title}
                 </span>
               )}
-              {participants && participants.nodes.map((user, key) => (
-                <img className="participants" key={key} src={user.avatarUrl} title={user.login} />
-              ))}
+              <div className="avatar-stack">
+                {participants && participants.nodes.map((user, key) => (
+                  <img className="avatar" key={key} src={user.avatarUrl} title={user.login} />
```

Comment:
a valid URL format may contain space character (bad practice, but happens) as well.. I'd suggest adding the value for src within quotes.

Reply:
@sujitagarwal In browser it renders the URL in quotes

```html
<img class="avatar" src="https://avatars3.githubusercontent.com/u/67512?v=4" title="dougwilson">
```

Reply:
That's a browser specific feature, and quite unreliable when trying out with some unpopular ones

Reply:
@sujitagarwal React renders the same across all the browsers!

http://blog.matthewcheok.com/a-better-img-tag/

Reply:
Great.